Packs/Akamai_SIEM/Integrations/Akamai_SIEM/Akamai_SIEM.py

```diff
@@ -551,6 +553,618 @@ def decode_url(headers: str) -> dict:
     return decoded_dict
 
 
+############################################## Beginning of beta part ##############################################
+BETA_FETCH_EVENTS_MAX_PAGE_SIZE = 600000  # Allowed events limit per request.
+LOCKED_UPDATES_LOCK = None
+EVENTS_LS = ""
+MAX_ALLOWED_TASKS_SIZE = (10 ** 10)  # 10 GB
+
+
+async def get_events_with_offset_aiohttp(
+    self,
+    config_ids: str,
+    offset: str | None = '',
+    limit: int = 20,
+    from_epoch: str = '',
+    counter: int = 0
+) -> tuple[list[str], str | None]:
+    params: dict[str, int | str] = {
+        'limit': limit
+    }
+    if offset:
+        demisto.info(f"received {offset=} will run an offset based request.")
+        params["offset"] = offset
+    else:
+        from_param = int(from_epoch)
+        params["from"] = from_param
+        demisto.info(f"did not receive an offset. will run a time based request with {from_param=}")
+
+    url = "https://edl-viso-qb8hymksjijlrdzyknr7rq.xdr-qa2-uat.us.paloaltonetworks.com/xsoar/instance/execute/Generic_Webhook_instance_1/"
+    headers = {
+    'Authorization': 'Basic YTph',
+    }
+    demisto.info(f"Init session and sending request for the {counter} time.")
```

Comment:
would remove counter in favor of an ID (lets not use uuid, use a counter that resets at 100_000)